mne/viz/epochs.py

```diff
@@ -1020,7 +1044,7 @@ def _plot_vert_lines(params):
     epochs = params['epochs']
     if params['settings'][3]:  # if zeroline visible
         t_zero = np.where(epochs.times == 0.)[0]
-        if len(t_zero) == 1:
+        if len(t_zero) == 1:  # not True if tmin > 0
```

Comment:
previously if `if len(t_zero) == 0` then `t_zero` is turned to single value so it would have no len